src/main/java/com/conveyal/analysis/models/Bundle.java

```diff
@@ -104,6 +104,7 @@ public FeedSummary clone () {
         }
     }
 
+    // The first two PROCESSING_* values are essentially deprecated in favor of Task. Consider eliminating this field.
     public enum Status {
```

Comment:
I think we can eliminate `Status` here, and we should consider changing `statusText` to something it will more correctly represent -- `errors`.

Reply:
Or something similar

Reply:
I agree, I've been progressively removing updates to this field but didn't want to leave it out entirely in case UI or other components need it.